examples/ScatteringExample/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.12)
+project(scattering VERSION 0.1.0)
+
+# Compiler/build settings
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# CTest if we need it
+include(CTest)
+
+# Dependencies
+find_package(Geant4 REQUIRED)
```

Comment:
@yramachers Any objection to changing this to `find_package(Geant4 10.7 REQUIRED)`?

I'm fairly sure there's quite a few things (e.g. `#include "G4RunManagerFactory.hh"`) which require 10.7.

Reply:
No, 10.7 really is a minimum requirement. Didn't know one can require a version of Geant4 in the Cmake. That's fine.